corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -63,12 +63,12 @@ <h2 tabindex="0"><%- title %></h2>
     class="<% if (contentTag === 'td') { %>col-sm-6 <% } %>query-caption
            <% if (required) { %> required<% } %>
            <% if (errorMessage) { %> has-error<% } %>">
-    <div>
+    <div style="display: grid; grid-template-columns: auto auto">
```

Comment:
please no inline styles! 😅 between this an all the overrides in our stylesheets it's getting rowdy in webapps

Reply:
I for sure would move these to a class. But from our discussion on slack it would be even better to reuse existing classes (from bootstrap or otherwise). I just have not been able to get it to work/look right with those.

Reply:
@biyeun I found `.justify-content-between` in bootstrap 5 so I added it to bootstrap5-temp.less as I did with `.align-items-center`. I think that is a fairly clean solution.

Reply:
@biyeun could you approve?